### ISSUE TITLE
ScrollView props stickyHeaderIndices press event not working

### ISSUE THREAD

**Issue by baofen14787** [facebook/react-native#696]
```
<ScrollView stickyHeaderIndices={[4]} >
    <View />
    <View />
    <View />
    <View />
    <Tabs />
    <View />
    <View />
    <View />
    <View />
    <View />
    <View />
</ScrollView>  
```

and <Tbas /> have onPress event in childern.

when scrollto Tabs sticky , onPress event not work.

**Comment by CYJB** [comment]
Scroll is also not working. It looks like the child has been frozen.

**Comment by iostalk** [comment]
Help!Help!

**Comment by brentvatne** [comment]
This is fixed now: #1415 :smile:
